pages/collection/[collection_id].js

```diff
@@ -303,6 +305,12 @@ const CollectionPage = ({ collectionId, collection, serverQuery }) => {
 		}
 	}
 
+	const onClickDisplay = (typeDisplay) => {
+		setTokens([])
+		fetchData()
+		setDisplay(typeDisplay)
+	}
+
```

Comment:
since ini ngeset tokensnya jadi empty array lagi, berarti ngefetch dari 0. `idNext`, `lowestPriceNext`, `updatedAtNext` sama `hasMore` nitip di update juga ya ke initial valuenya